charts/seed-controlplane/charts/deprecated-network-policies/templates/deprecated-apiserver-network-policy.yaml

```diff
@@ -0,0 +1,14 @@
+# DEPRECATED. DO NOT CREATE THIS CHART.
```

Comment:
Why can't we remove the chart entirely now?

Reply:
OK, I saw below what you have built. The `DeleteManifest` function is OK, but we can't be sure that everything is deleted if we use it for cleanup. To be really sure we need the same `values.yaml` that we used during deployment of the chart, and this we don't have anymore (there might be resources that are only deployed if a certain value is set, for example).
I'd prefer to remove the chart now, and to not use the `DeleteManifest` function for cleanup, but to directly delete the old network policies (as we anyway know their names).

Reply:
> but we can't be sure that everything is deleted if we use it for cleanup.

Care to elaborate?

This chart is now calling more or less `DELETE` 3 times for those policies. If they do not exists, then it's a no problem in the first place.

Reply:
I know that it works for this simple chart. However, having the `DeleteChart` function does not work in all cases.
We have a lot of charts that contain templates which are only rendered if a certain value in the `values.yaml` is set. The `DeleteChart` function is rendering the chart with the default values. The chances are very high that those templates might be missed. As a consequence, they are not deleted and remain orphan in the system. The only way to be sure that everything is cleaned up correctly is to use the same `values.yaml` for the deletion that were used for the deployment. However, we don't have these values anymore.

Reply:
I've removed this DeleteChart function